b2sdk/account_info/in_memory.py

```diff
@@ -133,13 +133,13 @@ def get_account_id(self):
         return self._account_id
 
     @_raise_missing_if_result_is_none
-    def get_account_id_or_app_key_id(self):
+    def get_application_key_id(self):
         """ 
-        Returns the account id or key id used to authenticate
+        Returns the application key id used to authenticate
 
         :rtype: str
         """
-        return self._account_id_or_app_key_id
+        return self._app_key_id
```

Comment:
The variable containing it is sometimes called:
* application_key_id
* _app_key_id / app_key_id

I think there is no need to shorten it (we all use variable completion in our IDEs anyway), but there is a great benefit of having it consistently named across the entire library. When the argument is renamed in each layer, it becomes difficult to read the code.